tests/common.go

```diff
@@ -2912,6 +2914,198 @@ func DeleteBucket(provider string, bucketName string) {
 	})
 }
 
+//HaIncreaseRebootTargetNode repl increase and reboot target node
+func HaIncreaseRebootTargetNode(event *EventRecord, ctx *scheduler.Context, v *volume.Volume, storageNodeMap map[string]node.Node) {
+
+	Step(
+		fmt.Sprintf("repl increase volume driver %s on app %s's volume: %v and reboot target node",
+			Inst().V.String(), ctx.App.Key, v),
+		func() {
+			currRep, err := Inst().V.GetReplicationFactor(v)
+
+			if err != nil {
+				err = fmt.Errorf("error getting replication factor for volume %s, Error: %v", v.Name, err)
+				logrus.Error(err)
+				UpdateOutcome(event, err)
+				return
+			}
+			//if repl is 3 cannot increase repl for the volume
+			if currRep == 3 {
```

Comment:
This case will be true for apps which is using repl 3. Can we have warning here?

Reply:
fixed